internal/pkg/template/templates/addons/aurora/env/serverlessv2.yml

```diff
@@ -0,0 +1,178 @@
+      MasterUsername:
+        !Join [ "",  [ {{`'{{resolve:secretsmanager:'`}}, !Ref {{logicalIDSafe .ClusterName}}AuroraSecret, ":SecretString:username}}" ]]
+      MasterUserPassword:
+        !Join [ "",  [ {{`'{{resolve:secretsmanager:'`}}, !Ref {{logicalIDSafe .ClusterName}}AuroraSecret, ":SecretString:password}}" ]]
+      DatabaseName: !Ref {{logicalIDSafe .ClusterName}}DBName
+      {{- if eq .Engine "MySQL"}}
+      Engine: 'aurora-mysql'
+      EngineVersion: '8.0.mysql_aurora.3.02.0'
+      {{- else}}
+      Engine: 'aurora-postgresql'
+      EngineVersion: '14.4'
+      {{- end}}
+      DBClusterParameterGroupName: {{- if .ParameterGroup}} {{.ParameterGroup}} {{- else}} !Ref {{logicalIDSafe .ClusterName}}DBClusterParameterGroup {{- end}}
+      DBSubnetGroupName: !Ref {{logicalIDSafe .ClusterName}}DBSubnetGroup
+      Port: {{if eq .Engine "MySQL"}}3306{{else}}5432{{end}}
+      VpcSecurityGroupIds:
+        - !Ref {{logicalIDSafe .ClusterName}}DBClusterSecurityGroup
+      ServerlessV2ScalingConfiguration:
+        # Replace "All" below with "!Ref Env" to set different autoscaling limits per environment.
+        MinCapacity: !FindInMap [{{logicalIDSafe .ClusterName}}EnvScalingConfigurationMap, All, DBMinCapacity]
+        MaxCapacity: !FindInMap [{{logicalIDSafe .ClusterName}}EnvScalingConfigurationMap, All, DBMaxCapacity]
+  
+  {{logicalIDSafe .ClusterName}}DBWriterInstance:
+    Metadata:
+      'aws:copilot:description': 'The {{logicalIDSafe .ClusterName}} Aurora Serverless v2 writer instance'
+    Type: 'AWS::RDS::DBInstance'
+    Properties:
+      DBClusterIdentifier: !Ref {{logicalIDSafe .ClusterName}}DBCluster
+      DBInstanceClass: db.serverless
+      {{- if eq .Engine "MySQL"}}
+      Engine: 'aurora-mysql'
+      {{- else}}
+      Engine: 'aurora-postgresql'
+      {{- end}}
+      PromotionTier: 1
+      AvailabilityZone: !Select
+        - 0
+        - !GetAZs
+          Ref: AWS::Region
+
+  {{logicalIDSafe .ClusterName}}SecretAuroraClusterAttachment:
+    Type: AWS::SecretsManager::SecretTargetAttachment
+    Properties:
+      SecretId: !Ref {{logicalIDSafe .ClusterName}}AuroraSecret
+      TargetId: !Ref {{logicalIDSafe .ClusterName}}DBCluster
+      TargetType: AWS::RDS::DBCluster
+
+Outputs:
+  {{logicalIDSafe .ClusterName}}Secret: # injected as {{envVarSecret .ClusterName | toSnakeCase}} environment variable by Copilot.
+    Description: "The JSON secret that holds the database username and password. Fields are 'host', 'port', 'dbname', 'username', 'password', 'dbClusterIdentifier' and 'engine'"
+    Value: !Ref {{logicalIDSafe .ClusterName}}AuroraSecret
+    Export:
+      Name: !Sub ${Env}-{{logicalIDSafe .ClusterName}}AuroraSecret
+  {{logicalIDSafe .ClusterName}}SecurityGroup:
+    Description: "The security group to attach to the workload."
+    Value: !Ref {{logicalIDSafe .ClusterName}}WorkloadSecurityGroup  
+    Export:
+      Name: !Sub ${Env}-{{logicalIDSafe .ClusterName}}SecurityGroup
```

Comment:
Do we want to namespace the stack output such as `copilot-${App}-${Env}-{{logicalIDSafe .ClusterName}}SecurityGroup`

Reply:
What do you think of `${App}-${Env}-{{logicalIDSafe .ClusterName}}SecurityGroup` instead so that people can write in manifest:
```
security_groups: 
  - from_cfn: ${COPILOT-APPLICATION-NAME}...
```
instaed of 
```
security_groups: 
  - from_cfn: copilot-${COPILOT-APPLICATION-NAME}...
```